src/Error/ErrorLoggerInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.1.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Error;
+
+use Psr\Http\Message\ServerRequestInterface;
+use \Throwable;
+
+/**
+ * Interface for error logging handlers.
+ *
+ * Used by the ErrorHandlerMiddleware and global
+ * error handlers to log exceptions and errors.
+ */
+interface ErrorLoggerInterface
```

Comment:
It would have been nice if we could just use or extend `LoggerInterface` (as we did for `Connection`). But since existing `ErrorLogger::log()` has a different signature it would be a BC break.

Reply:
A cleanup for 5.x maybe then :)

Reply:
Yeah, we can't do that today unfortunately.